views/views.py

```diff
@@ -147,6 +262,50 @@ def create(self, request, *args, **kwargs):
             comment = serializer.save(commenter=person)
             query.comments.add(comment)
             query.save()
+
+            # Notifications start
+
+            url = f'/helpcentre/issue/{query_id}'
+            category_obj = get_base_category()
+            person_ids = list(query.assignees.all().values_list(
+                'person_id',
+                flat=True
+            ))
+            person_ids = [
+                id for id in person_ids if id != person.id
+            ]
+            template = (
+                f'{person.full_name} added a comment on '
+                f'{query.uploader.full_name}\'s query {query.title}'
+            )
+
+            # For fellow assignees
+            if person_ids:
+                push_notification(
+                    template=template,
+                    category=category_obj,
+                    web_onclick_url=url,
+                    persons=person_ids,
+                    has_custom_users_target=True
+                )
+
+            # For the uploader
+            if query.uploader_id != person.id:
+                uploader_id = query.uploader_id
+                template = (
+                    f'{person.full_name} added a '
+                    f'comment on your query {query.title}'
+                )
+                push_notification(
+                    template=template,
+                    category=category_obj,
+                    web_onclick_url=url,
+                    person=uploader_id,
+                    is_personalised=True
+                )
+
+            # Notifications end
```

Comment:
Bad practice to mark block end. Instead mark the start of the next block if that might be confusing.

Reply:
Okay, thanks I was not aware of that.